homeassistant/components/decora_wifi/common.py

```diff
@@ -0,0 +1,144 @@
+"""Common code for decora_wifi."""
+
+import logging
+from typing import Dict
+
+# pylint: disable=import-error
+from decora_wifi import DecoraWiFiSession
+from decora_wifi.models.person import Person
+from decora_wifi.models.residence import Residence
+from decora_wifi.models.residential_account import ResidentialAccount
+
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, LIGHT_DOMAIN, PLATFORMS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class DecoraWifiPlatform:
```

Comment:
This class contains only static methods, so is a class really needed?

Additionally, if you actually would remove the staticmethod decorators and instantiate this class; it would save a lot of juggling with the config entry data (e.g., as fetched on config entry unload to logout).

Reply:
My thought is that wrapping all of the calls to the external API in a utility class allows each platform.py module to simply `from .common import DecoraWifiPlatform` instead of having to import each individual function it needs.  Seems this is not as common in Python as it is in other languages I've worked in, though.

Reply:
Sure, but right now, it is constantly building a new session. Why not create a session and store a handler to that object instead? All subsequent calls will already have a session, and the logins, removing the need to do it again every single call.

Reply:
Took a swing at this one yesterday; I think I've addressed the issue of just building the session once and continuing to use it now.  Instantiating the class had a few side benefits to the implementation as well. Still not sure if the way it's being stored is the best though; I had thought to just throw it into the config entry data at some point during the development but I got an error in my log basically saying that `DecoraWiFiSession` (the session class from the upstream API) is not serializable which is why I went this direction.

I know the term handler with respect to files or events, but I'm not familiar with the term 'handler to [an] object'; is that referring to like a proxy object?